ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Start script for applications.developer.ch.gov.uk
+
+PORT=3000
+
+source "server/config/.env"
+export NODE_PORT=${PORT}
+node server.js
```

Comment:
The Dockerfile refers to a build arg for PORT but it's hard coded in this script. Also `NODE_PORT` is defined as a build arg but is set to be the same hard-coded value as `PORT` - should this be dependent on an environment variable instead, with the ARG in the Dockerfile assigned to ENVs?